### DISCUSSION TITLE
My personal theme

### DISCUSSION THREAD

**Discussion by HongYue1** [kimlimjustin/xplorer#267] [Show and tell]
Hi, I made a theme which suits my taste and I thought I might share it with you guys in case anyone likes it.

<a href="https://mega.nz/file/6OYxDZzA#tQkmT3KcoMYOs7NLbA1ISkSBAF-lS9guao8NFYXBJ2U">Download</a>

<img width="1280" alt="2022-10-02_19-23" src="https://user-images.githubusercontent.com/87040288/193467576-dc6ef932-45a3-4991-8ac9-88189cdcb593.png">
<img width="1280" alt="2022-10-02_19-23_1" src="https://user-images.githubusercontent.com/87040288/193467588-b1448e5b-3f44-4606-a17c-c5d694dff716.png">

<a href="https://mega.nz/file/CORyjDTR#mW1MGqtgu0T-qPE1omQoiO8C9niQmh_9pZkLdIbHUzY">Darker version.</a>

<img width="1280" alt="2022-10-03_19-44" src="https://user-images.githubusercontent.com/87040288/193644141-8497009d-5221-46f0-b0ac-8ecae282bb57.png">

**Comment by HongYue1** [comment]
By the way, it's not really important, but this line is missing in the theme template when creating a new theme: 
`"root#sidebarResizer.background": "#2c2c58d9",`

**Reply by lg-nineoneninethreenine** [reply]
Well, is it required?

**Reply by HongYue1** [reply]
It's used to change the color of the separation line between the favorites sidebar and the main window, so maybe yes.

![image](https://user-images.githubusercontent.com/87040288/193573881-c8b0cc04-4c3a-40ab-84a0-d06726235caa.png)

**Reply by kimlimjustin** [reply]
Oh, that's a good spot, however, that's in purpose to designed for the default theme, thanks!

**Comment by kimlimjustin** [comment]
Wow, that's amazing!
